packages/contracts-bedrock/scripts/forge-test-names.ts

```diff
@@ -0,0 +1,81 @@
+import fs from 'fs'
+import path from 'path'
+
+const testPath = './contracts/test'
+const testFiles = fs.readdirSync(testPath)
+
+// Given a test function name, ensures it matches the expected format
+const handleFunctionName = (name: string) => {
+  if (!name.startsWith('test')) {
+    return
+  }
+  const parts = name.split('_')
```

Comment:
Minor nitpick that I would prefer each of these checks to be separated by a newline

Reply:
N/A as I'm incorporating this suggestion: https://github.com/ethereum-optimism/optimism/pull/4124/files#r1036687919